### ISSUE TITLE
Syntax highlighting

### ISSUE THREAD

**Issue by codingtwinky** [rtfpessoa/diff2html-nodejs#7]
It would nice to have syntax highlighting.

I took a look at the existing opensource solutions to do this and there are couple of good libraries to do this with.  [prism](http://prismjs.com/) and [highlight](https://highlightjs.org/).  They both work similarly as in they will look for `<pre> <code> ... </code> </pre>` in html code and apply their function on it's contents.  I think there is two ways to use these libraries.  

1. Find a functions to generate output given code and use it in current generate html loop.
2. Modify current generate html to generate line number and place diff in `<pre> <code> ... </code> </pre>` without any rows for easy parsing for these libraries.

i.e. for normal diff
```
<line#><line#> |--------------|
<line#><line#> |              |
<line#><line#> |  code block  |
<line#><line#> |              |
<line#><line#> |______________|
```

I tried to do (1) above but I wasn't so successful as both of the libraries seems to be off.  

Also there are possible complication as '+' and '-' characters are included within code as result of git diff.

**Comment by rtfpessoa** [comment]
@codingtwinky if you look at this example: http://rtfpessoa.github.io/diff2html/
I already use highlightjs and is pretty easy to integrate. But is an addon you can add separately.
I use the extension to choose the language of the files but I also noticed some problems with the `+` and `-`

**Comment by rtfpessoa** [comment]
https://github.com/rtfpessoa/diff2html/commit/e598a81aa242dbe2ac361b841f377f4fa78b9934 should fix any highlight issues

**Comment by rtfpessoa** [comment]
fixed on #8
